docs_user/assemblies/assembly_prepare_director_operator_and_rhoso_for_adoption_process.adoc

```diff
@@ -0,0 +1,424 @@
+
+. Log in to the {OpenShift } environment where {OpenStackShort} OSPdO is deployed and change to the project that hosts your {OpenStackShort} deployment:
++
+----
+$ oc project openstack
+----
+
+. Define the common environment variables:
+----
+PASSWORD_FILE="tripleo-passwords.yaml"
+export PASSWORD_FILE
+
+OSP18_NAMESPACE="osp18"
+export OSP18_NAMESPACE
+
+OSPDO_NAMESPACE="openstack"
+export OSPDO_NAMESPACE
+
+OS_CLIENT="oc exec -c openstackclient -t openstackclient -- "
+export OS_CLIENT
+
+CONTROLLER_SSH="oc rsh -n $OSPDO_NAMESPACE -c openstackclient openstackclient ssh controller-0.ctlplane"
+export CONTROLLER_SSH
+
+CONTROLLER1_SSH="oc -n $OSPDO_NAMESPACE rsh -c openstackclient openstackclient ssh controller-0.ctlplane"
+export CONTROLLER1_SSH
+CONTROLLER2_SSH=
+export CONTROLLER2_SSH
+CONTROLLER3_SSH=
+export CONTROLLER3_SSH
+
+OSPDO_INTERNAL_API_NET="internalapi"
+export OSPDO_INTERNAL_API_NET
+
+STORAGE_CLASS=$(oc get -n $OSPDO_NAMESPACE pvc openstackclient-hosts -o jsonpath='{.spec.storageClassName}')
+export STORAGE_CLASS
+----
+
+. Get the name of the node where the current {OpenStackShort} Controller virtual machine (VM) is running:
++
+----
+$ CONTROLLER_NODE=$(oc -n $OSPDO_NAMESPACE get vmi -ojson | jq -r '.items[0].status.nodeName')
+export CONTROLLER_NODE
+----
+
+
+. If you are using a combination master/worker cluster with a high availability OSPdO environment,
+configure OSPdO to use a single {OpenStackShort} Controller node:
+
+.. Disable stonith:
++
+----
+$ stonith_resources=$(sudo pcs stonith status|grep Started|awk '{print $2}')
+for in in stonith_resources ;do CONTROLLER1_SSH sudo pcs stonith disable ;done
+----
+.. Put non-Controller nodes in maintenance:
++
+----
+$CONTROLLER1_SSH sudo pcs node maintenance <controller-1> <controller-2>
+----
++
+* Replace `<controller-1>` and `<controller-2>` with the names of the Controller nodes in your environment.
+
+.. Update the quorum for single node:
++
+----
+$CONTROLLER1_SSH sudo systemctl stop corosync
+$CONTROLLER2_SSH sudo systemctl stop corosync
+$CONTROLLER3_SSH sudo systemctl stop corosync
+$CONTROLLER1_SSH sudo pcs quorum update last_man_standing=1 wait_for_all=1
+$CONTROLLER1_SSH sudo systemctl restart corosync
+----
+.. Restart the Pacemaker cluster:
++
+----
+$CONTROLLER1_SSH sudo pcs cluster stop
+$CONTROLLER1_SSH sudo pcs cluster start
+----
+.. Remove the OSPdO node network configuration policies from the other two nodes that are not running the Controller VM by placing a `nodeSelector` on the {OpenShiftShort} node that contains the Controller VM:
++
+----
+$ for i in br-ctlplane br-ex br-osp; do oc patch osnetconfig openstacknetconfig --type json -p '[{"op": "replace", "path": "/spec/attachConfigurations/'$i'/nodeNetworkConfigurationPolicy/nodeSelector", "value": {"kubernetes.io/hostname": "'$CONTROLLER_NODE'"}}]'; done
+----
+
+== Preparing RHOSO for adoption
+
+When deploying a RHOSO env. usign the same master nodes, make sure
+to take the following steps:
```

Comment:
```suggestion
When you deploy a {rhos_long} environment by using two Red Hat OpenShift Container Platform master nodes, you must perform the following actions:
* Create a new namespace for {rhos_acro} installation.
* Create custom node network configuration policies.
* Create  `NetworkAttachmentDefinition` custom resources for each isolated network.
* Install {rhos_acro} operators.
```
Does "the same" refer to the two OpenShift master nodes that were used in the previous procedure?
Also, does the list I added accurately summarize what the customer is going to do in this procedure? (We don't have to list everything; it's just a high-level summary).

Reply:
yes , 'the same' is those same controllers used previously by th ospdo env.
and yes, that is a good high level summary